### ISSUE TITLE
Invalid Built Version 0.9.4

### ISSUE THREAD

**Issue by Grabauskas** [iamhosseindhv/notistack#185]
It seems like version 0.9.4 has some trash or not fully built.

https://unpkg.com/browse/notistack@0.9.4/build/SnackbarContainer/SnackbarContainer.js
```js
25  |   var _constants = require('../utils/constants');
... |
50  |   renderVariant: _propTypes2.default.oneOf(Object.keys(_constants.RENDER_VARIANTS)),
```

This file using `RENDER_VARIANTS` from constants, but it's not defined.
https://unpkg.com/browse/notistack@0.9.4/build/utils/constants.js

**Comment by iamhosseindhv** [comment]
Thanks for reporting this @Grabauskas . This is trash and I'll publish thefix by the end of today. Is this causing any issues while using the library?

**Comment by Grabauskas** [comment]
```json
map: {
   "build/SnackbarContainer": "build/SnackbarContainer/index.js"
}
```

So, when I requiring `build/SnackbarContainer` then old files was loaded.

**Comment by Grabauskas** [comment]
Yes, in JSPM this generated map line is invalid:
```json
map: {
   "./build/SnackbarContainer": "./build/SnackbarContainer/index.js"
}
```
But I already fixed that with overwrites, removing this line.

**Comment by iamhosseindhv** [comment]
This is fixed in `v0.9.5`. would be good if you could confirm.

**Comment by Grabauskas** [comment]
Thank you, now it works. 👍
